lib/hyrax/hirmeos/work_factory.rb

```diff
@@ -6,8 +6,8 @@ def self.for(resource:)
     work = Hyrax::Hirmeos::Client::Work.new
     work.title = resource.title
     work.uri = [{ uri: resource_url(resource), canonical: true },
-                { uri: resource.id }]
-    work.type = "other" # Need to map Hyrax work type to configured work types in HIRMEOS
+                { uri: "urn:uuid:#{resource.id}" }]
+    work.type = "repository-work"
```

Comment:
Hmmm, good point. It is just a database value, but one I added specifically to our own HIRMEOS translator instance - so this would not work on an "out of the box" HIRMEOS installation. :thinking: 

So, yes, either making a note or making this value confugurable (and leaving the default as "other") might not be a bad idea...